addons/supplies/config.cpp

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {
+            QGVAR(Launchers),
+            QGVAR(Engineer),
+            QGVAR(Medic),
+            QGVAR(Recruit)
+        };
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"tac_main", "cba_main", "ace_medical_treatment"};
```

Comment:
```suggestion
        requiredAddons[] = {"tac_main", "ace_medical_treatment"};
```
Already required by ACE.